.github/workflows/crucible-ci.yaml

```diff
@@ -2,8 +2,6 @@ name: crucible-ci
 
 on:
   # run on push or pull request events for the main branch only
-  push:
-    branches: [ main ]
```

Comment:
maybe for the ci repo itself it's a good idea to keep it.... Let's say you have a chicken and egg problem, where the ci config only works after merging... if you run after merging you make sure the test ran fine.

Reply:
That is a good point...I had sorta convinced myself that changes that were part of the PR were being tested as well but I could be wrong...I've tested so many scenarios I may be mistaken about that combination.  I'm ok with leaving it for now and see how it goes.  So we should just close this one?